src/test/java/ua/javaPro/berezhnoy/hillelHomeWork4/ArrayValueCalculatorTest.java

```diff
@@ -0,0 +1,54 @@
+package ua.javaPro.berezhnoy.hillelHomeWork4;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+class ArrayValueCalculatorTest {
+    String[][] array = {{"1", "2", "3", "4"},
+            {"5", "6", "7", "8"},
+            {"9", "10", "11", "12"},
+            {"13", "14", "15", "16"}};
+
+
+    @Test
+    public void arrayTrue() {
+        assertDoesNotThrow(() -> ArrayValueCalculator.doCalc(array));
+    }
+
+    @Test
+    void arraySize() {
```

Comment:
```suggestion
    void shouldThrowExceptionForWrongArraySize() {
```
так, довга назва, але з неї одразу зрозуміло що тест буде робити.